scripts/bootstrap-aio.yml

```diff
@@ -147,7 +147,6 @@
         - name: "env.d/magnum.yml"
         - name: "env.d/memcache.yml"
         - name: "env.d/neutron.yml"
-        - name: "env.d/nova.yml"
```

Comment:
This will need to stay for now until https://github.com/rcbops/rpc-openstack/pull/2375 is finalised and merged.